submissions/ivan-chukhalo/pop-up_hooli/index.html

```diff
@@ -0,0 +1,112 @@
+    <link rel="stylesheet" href="./css/style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <a href="#" class="header__link link">
+        <img class="header__logo" src="./assets/logo.png" alt="Go to the home page of the site" />
+      </a>
+      <nav class="header__nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a href="#" class="nav__link link">mail</a>
+          </li>
+          <li class="nav__list-item">
+            <a href="#" class="nav__link link">images</a>
+          </li>
+          <li class="nav__list-item nav__popup-btn">
+            <input type="checkbox" class="checkbox" id="checkbox_popup"/>
+            <label class="nav__label" for="checkbox_popup">
+              <img src="./assets/icons/icon_popup.png" alt="Open the list with other Hooli services" class="nav__icon" />
+            </label>
+            <div class="popup-wrapper">
+              <ul class="popup">
+                <li class="popup__item">
+                  <a href="#" class="popup__link link icon_youtube">tube</a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__link link icon_docs">docs</a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__link link icon_chat">chat</a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__link link icon_label">label</a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__link link icon_yellow">yellow</a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__link link icon_drive">drive</a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__link link icon_flower">flower</a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__link link icon_desk">desk</a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__link link icon_calendar">calendar</a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__link link icon_musiem">musiem</a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__link link icon_apps">apps</a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__link link icon_excel">tables</a>
+                </li>
+                <li class="popup__item popup__item_more">
+                    <input type="checkbox" class="checkbox checkbox_more" id="checkbox_more" />
+                    <label class="popup__item popup__more" for="checkbox_more">More</label>
```

Comment:
Logically this button must be hidden when the second part of the popup is open
![image](https://user-images.githubusercontent.com/16196199/185790844-c7f367ad-6329-462f-bbdd-b1b4a245c3d6.png)